app/models/katello/activation_key.rb

```diff
@@ -290,7 +290,7 @@ def check_cves
       property :name, String, desc: 'Returns the name of the Activation Key.'
     end
     class Jail < ::Safemode::Jail
-      allow :name
+      allow :name, :content_view_environment_labels, :multi_content_view_environment?
```

Comment:
I suppose the `:multi_content_view_environment?` was added just out of future helpfulness to users?

Reply:
Yeah. Figured it may save an RFE in the future.